docs/guides/fabric_synchronization_guide.md

```diff
@@ -118,6 +118,11 @@ of Ecosystem 1 has successfully paired with Ecosystem 2 on Endpoint 2:
 >>> A new device is added on Endpoint 2.
 ```
 
+Note: We only need to add the local bridge to the ecosystem to trigger the
+Fabric Sync Setup process. In the example above, the Fabric Sync Setup process
```

Comment:
Does it make sense to describe what "the Fabric Sync Setup process" is because that might also no be clear.